packages/client/src/checkout/__fixtures__/postCheckoutOrderContext.fixtures.ts

```diff
@@ -0,0 +1,24 @@
+import { rest, type RestHandler } from 'msw';
+import type { PostCheckoutOrderContextResponse } from '../types/index.js';
+
+const path = '/api/checkout/v1/orders/:checkoutOrderId/contexts';
+
+const fixtures = {
+  success: (response: PostCheckoutOrderContextResponse): RestHandler =>
+    rest.post(path, (_req, res, ctx) =>
+      res(
+        ctx.status(200),
```

Comment:
Probably a nitpick but the response status is 201

Reply:
done, can you check please?